src/cli/common.js

```diff
@@ -305,20 +305,28 @@ function cli(api){
     function readConfigData(config) {
         var data = readConfigFile(config),
             json,
+            optionName,
+            optionValue,
+            args,
             options = {};
         if (data) {
             if (data.charAt(0) === "{") {
                 try {
                     json = JSON.parse(data);
                     data = "";
-                    for (var optionName in json) {
+                    for (optionName in json) {
```

Comment:
Because I added `optionName` to the `var` declarations at the top of the function.